src/notification/notification.service.ts

```diff
@@ -15,8 +16,14 @@ export default class NotificationService {
         senderUserID: string,
         receiverUserID: string
     ): Promise<void> {
+        const destination: string | undefined = !this.isValidObjectId(
+            senderUserID
+        )
+            ? String(EXTERNAL_DESTS.TOMCAL)
+            : undefined;
+
```

Comment:
What are you doing here?

Reply:
The user service accepts an argument that tells which `destination` it should get the user from (practically it tells the user service to get the user from the Kartoffel or the Phonebook).
So, `destination` is defaulted on Kartoffel (therefore the `undefined`) but if the `senderUserID` is not type of Mongodb ObjectId, it tells that it is not a Kartoffel ObjectId so we should take the user from the Phonebook instead of Kartoffel.

Reply:
Good question